datasets.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pandas as pd
+from numpy.random import uniform, exponential, triangular
+
+SYNTHETIC_DATASET_TYPES = ["uniform", "linear", "exponential"]
+
+
+def generate_uniform(n):
+    return np.ones(n)
```

Comment:
It has to do with their definition of a uniform dataset. I removed it now, note that this does not produce any error when running python code. If you have problems running the code it might help to actually report on what kind of errors you are facing. The only response I can give here is _it works fine on my machine_.